src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -55,4 +67,20 @@ public Post getRelatedPost() {
     public void setRelatedPost(Post relatedPost) {
         this.relatedPost = relatedPost;
     }
+    
+    public User getRelatedUser() {
+        return relatedUser;
+    }
+
+    public void setRelatedUser(User relatedUser) {
+        this.relatedUser = relatedUser;
+    }
+
+    public List<PostLike> getListOfLikes() {
+        return listOfCommentLikes;
+    }
+
+    public void setListOfLikes(List<PostLike> listOfCommentLikes) {
+        this.listOfCommentLikes = listOfCommentLikes;
+    }
```

Comment:
Please remove this we will implement later

Reply:
it's been removed